pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
-name = "meltano.edk"
-version = "0.1.0"
+name = "meltano-edk"
```

Comment:
@WillDaSilva what do you think of this change? Not sure why Florian went for a dot here, but I haven't seen any packages in pypi following this convention, which makes me think it isn't allowed 🤔

Reply:
I'm pretty sure it will break if there is no dot there. The dot makes this into a namespace package.

Reply:
Python's standard around package names allows for dots in namespace packages (and, confusingly, non-namespace packages too), but they'll be normalized to `-` in most cases.

Reply:
Oh yeh, spotted the `meltano/edk` dir structure. I put the dot back, but if it causes me publishing trouble I'll flatten the dir 🤏😅 We don't use namespace packages anywhere else AFAIK and I don't see the particular benefit here.

Reply:
> if it causes me publishing trouble I'll flatten the dir

It won't, and you can't. Its import path is `meltano.edk`. Flattening the dir makes it something else. Everything that depends on it would break.